cpp/ql/src/Architecture/General Namespace-Level Information/CyclicNamespaces.qhelp

```diff
@@ -1,25 +0,0 @@
-<!DOCTYPE qhelp PUBLIC
-  "-//Semmle//qhelp//EN"
-  "qhelp.dtd">
-<qhelp>
-
-
-<overview>
-<p>This query shows namespaces that cyclically depend 
-on one another.</p>
-
-<p />
-
-</overview>
-<section title="How to Address the Query Results">
-<p>If there are cyclic dependencies between packages, they cannot be developed and tested independently. It is thus preferable to
-eliminate such cycles from the program.</p>
```

Comment:
This seems to contain some information, but were we going to deprecate these Architecture queries anyway? @geoffw0?

Reply:
These 'Architecture' queries have nothing to do with the 'Architect' product as far as I'm aware.  It's an unfortunate naming crossover.  I doubt the 'Architecture' queries get a lot of use and I don't think they've been touched for years - but I'm not aware of any specific plans to deprecate them.

Reply:
Also since many of them are unusual query kinds (`graph`, `treemap`) that are not supported by LGTM, I think it's unlikely we'll want to do anything with them soon.

Reply:
I agree that this one is rather borderline. In that case, I'll revert this deletion and update the section titles to meet current expectations. After that, will it be okay to publish it as-is or should it be added to the list for exclusion?

Reply:
I think it's fine.